server/api/users/signals.py

```diff
@@ -0,0 +1,17 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.conf import settings
+from api.users.models import User
+from django.core.mail import EmailMessage
+
+
+@receiver(post_save, sender=User)
+def send_new_user_to_admin(sender, instance, created, **kwargs):
+    if created:
+        email = EmailMessage(
+            'A User has registered with CSF',
+            '{} has registered with CSF'.format(instance.first_name + " " + instance.last_name),
```

Comment:
"Stride For Education: New User Registered"

and


'{} has registered an account on Stride For Education'